pvlib/pvsystem.py

```diff
@@ -1373,11 +1371,12 @@ class FixedMount(AbstractMount):
         Azimuth angle of the module surface. North=0, East=90, South=180,
         West=270. [degrees]
 
-    racking_model : str, optional
-        Valid strings are 'open_rack', 'close_mount', and 'insulated_back'.
-        Used to identify a parameter set for the SAPM cell temperature model.
+    racking_model : str, optional, default None
+        Valid strings are `open_rack`, `close_mount`, `freestanding`, or
+        `insulated_back`.
+        Used to identify a parameter set for the cell temperature model.
 
-    module_height : float, optional
+    module_height : float, optional, default None
```

Comment:
Same here -- either `optional` or `default None` (not both), depending on whether `None` is meaningful.